platform-hub-api/Dockerfile

```diff
@@ -1,22 +1,46 @@
-FROM ruby:2.3.3
-
-RUN apt-get update -y && apt-get -u upgrade -y
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" >> /etc/apt/sources.list.d/pgdg.list \
-    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
-    && apt-get update && apt-get install -y libpq-dev
-
-WORKDIR /app
+FROM quay.io/ukhomeofficedigital/centos-base
 
+ENV RUBY_VERSION 2.3.3
 ENV RAILS_ENV production
 ENV RACK_ENV production
 
-COPY Gemfile Gemfile.lock ./
-RUN bundle install --jobs 20 --retry 5 --deployment --without development test --no-cache
+# Update kernel-headers package
+RUN yum install -y wget \
+    && wget ftp://ftp.pbone.net/mirror/ftp.centos.org/7.3.1611/virt/x86_64/xen-46/kernel-headers-4.9.31-27.el7.x86_64.rpm \
```

Comment:
Can we trust `pbone.net`? (It's been a while since I've used CentOS so I'm lacking knowledge here :))